### PR TITLE
Switch release to maven central

### PR DESCRIPTION
**Pull request by dougwettlaufer**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -5,7 +5,11 @@
     <artifactId>stargate</artifactId>
     <version>0.0.6-SNAPSHOT</version>
     <packaging>pom</packaging>
-    
+
+    <name>${project.groupId}:${project.artifactId}</name>
+    <description>Core dependencies for Stargate</description>
```
Comment (dimas-b) [inline pom.xml:10]: `Core dependencies` -> `Core modules`? "Dependencies" sounds external to me :)

File: pom.xml [hunk 2]
```diff
@@ -89,6 +93,34 @@
...
+        <license>
+            <name>The Apache Software License, Version 2.0</name>
+            <url>http://www.apache.org/licenses/LICENSE-2.0.txt</url>
+        </license>
+    </licenses>
+
+    <scm>
+        <connection>scm:git:git://github.com/stargate/stargate.git</connection>
+        <developerConnection>scm:git:git@github.com:stargate/stargate.git</developerConnection>
+        <url>http://github.com/stargate/stargate/tree/master</url>
```
Comment (dimas-b) [inline pom.xml:107]: Why not just the base URL of `http://github.com/stargate/stargate`? ... just curious.

File: pom.xml [hunk 3]
```diff
@@ -89,6 +93,34 @@
...
+    </scm>
+
+    <developers>
+        <developer>
+            <name>Various</name>
```
Comment (dimas-b) [inline pom.xml:112]: `The Stargate Authors`? per our file header?

**Comment by dimas-b** [comment]
Is this driven by the `api-extensions` build failure or is there a deeper reason?.. just curious :)

File: pom.xml (hunk 2 above)
Comment (dougwettlaufer) [reply pom.xml:107]: Because I copy pasted the example from sonatype :)

**Review by dimas-b — APPROVED** [review]

**Comment by dougwettlaufer** [comment]
> Is this driven by the `api-extensions` build failure or is there a deeper reason?.. just curious :)

Well that is a contributing factor but Github packages is a hot mess. Plus I really don't like asking people to set up auth just to use our stuff.

**Pull request merged**